packages/store-ui/src/ProductQuantity/Title.tsx

```diff
@@ -8,4 +8,8 @@ export interface ProductQuantityTitleProps {
 export const ProductQuantityTitle: FC<ProductQuantityTitleProps> = ({
   variant = 'productDescription',
   children,
-}) => <Text variant={`${variant}.title`}>{children}</Text>
+}) => (
+  <Label htmlFor="product-quantity" variant={`${variant}.title`}>
```

Comment:
I think it's better to receive the `for` as a prop and specifying on the Store (both for the label and the input)

On a typical page, there's a possibility of having multiple Quantity Selector, what would cause an inconsistence (elements with the same id)

Reply:
hm, okay, I will do.